app/build.gradle

```diff
@@ -24,6 +24,9 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            create("customDebugType") {
```

Comment:
what is this for?

Reply:
i use it to run the debugger

Reply:
It is really necessary? I never need it in the past and I can debug the code. Try removing it and see the behavior please

Reply:
done